lib/internal/Magento/Framework/Model/EntitySnapshot/AttributeProviderInterface.php

```diff
@@ -12,8 +13,10 @@
 interface AttributeProviderInterface
 {
     /**
+     * Returns array of fields
+     *
      * @param string $entityType
-     * @return array
+     * @return string[]
      */
-    public function getAttributes($entityType);
+    public function getAttributes($entityType): array;
```

Comment:
Unfortunately, we are not allowed to change public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)